css/main.css

```diff
@@ -105,8 +105,11 @@ body {
 
 .logo-img {
   width: 200px;
+  height: 200px;
   display: block;
   margin: 20px auto;
+  overflow-anchor: none;
+  z-index: 1;
```

Comment:
Is z-index applying here ??
I think that it works only with `display: flex` and if we specify `position` attribute.